vllm/v1/spec_decode/eagle.py

```diff
@@ -118,7 +118,7 @@ def __init__(
         if self.uses_mrope:
             # M-RoPE need (3, max_num_tokens)
             self.mrope_positions = torch.zeros(
-                (3, self.max_num_tokens), dtype=torch.int64, device=device
+                (3, self.max_num_tokens + 1), dtype=torch.int64, device=device
```

Comment:
Context for this change is here:
https://github.com/vllm-project/vllm/blob/4c23690f43e51eccf6ce5866ac47adcf39215e4d/vllm/v1/worker/gpu_model_runner.py#L491